### ISSUE TITLE
Qt6 hidden UI text, webpage content displayed properly

### ISSUE THREAD

**Issue by blob42** [qutebrowser/qutebrowser#8347]
**Version info**:

- [version for hidden UI on qt6](https://bpa.st/4LIDO)
- [version OK on qt5](https://bpa.st/HLUHO)

Tried:
- qutebrowser 3.2.1-1
- qutebrowser-git 3.3.1.r52.gaf835c26a-1

**Does the bug happen if you start with `--temp-basedir`?**:
Yes

**Description**
After my upgrade on Arch, all the UI became hidden. Websites content is properly displayed.

- Other apps using QT6 work do not exhibit this issue
- Using QT5 with `--qt-wrapper PyQt5` resolves the issue

**How to reproduce**
![20241017_12h54m16s_grim](https://github.com/user-attachments/assets/bc39b07e-c1cb-4d4e-a215-b62ade76ffba)
![20241017_12h54m04s_grim](https://github.com/user-attachments/assets/016c40bd-5a5f-4a02-b73e-b06c9448d53b)

**Comment by blob42** [comment]
I managed to pinpoint the problem to `fontconfig`. I have the following config:


```xml
<?xml version="1.0"?>
<!DOCTYPE fontconfig SYSTEM "urn:fontconfig:fonts.dtd">
<fontconfig>
  <description>My Custom Font Config</description>

  <match target="font">
    <test name="weight" compare="more">
      <const>medium</const>
    </test>
    <edit name="autohint" mode="assign">
      <bool>false</bool>
    </edit>
  </match>

  <alias>
      <family>monospace</family>
      <prefer>
          <family>Tamzen</family>
          <family>Noto Color Emoji</family>
      </prefer>
  </alias>

</fontconfig>
```

If I disable it the problem is solved. This seems to only happen with Qt6

**Comment by blob42** [comment]
Problem solved: It was related to my font. Setting a different `fonts.default_family` solved the issue.

**Comment by bitraid** [comment]
You were using Tamzen (a bitmap font), and the newly released Qt 6.8, happens to have issues with bitmap fonts. See https://github.com/qutebrowser/qutebrowser/issues/8242#issuecomment-2416423234
